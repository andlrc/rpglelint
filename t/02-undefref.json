[
   {
      "column":1,
      "file":"t/02-undefref.rpgle",
      "line":5,
      "message":"'var3' undeclared",
      "severity":"warning"
   },
   {
      "column":7,
      "file":"t/02-undefref.rpgle",
      "line":6,
      "message":"'var4' undeclared",
      "severity":"warning"
   }
]
