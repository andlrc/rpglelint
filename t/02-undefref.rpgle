// rpglelint: -Wundefined-reference
dcl-s var1 int(10);
dcl-s var2 int(20);

var3 = 
      var4
      + var1 + var2;

dcl-proc my_proc;
  var1 = var2;
end-proc;
