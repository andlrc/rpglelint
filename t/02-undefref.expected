t/02-undefref.rpgle:5:1: warning: 'var3' undeclared [-Wundefined-reference]
 var3 = 
 ^
t/02-undefref.rpgle:6:7: warning: 'var4' undeclared [-Wundefined-reference]
       var4
       ^
